wiki/Main_Page/pt-br.md

```diff
@@ -1,61 +1,87 @@
 
+# Jogabilidade
 
-# [Editor de Beatmap](/wiki/Beatmap_Editor)
+[Modos de Jogo](/wiki/Game_Mode): [osu!](/wiki/Game_Mode/osu!) • [osu!taiko](/wiki/Game_Mode/osu!taiko) • [osu!catch](/wiki/Game_Mode/osu!catch) • [osu!mania](/wiki/Game_Mode/osu!mania)
 
-Seções: ([Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song Setup](/wiki/Beatmap_Editor/Song_Setup)) • Componentes: ([Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance Snap](/wiki/Beatmap_Editor/Distance_Snap)) • [Beatmapping](/wiki/Beatmapping) • [Dificuldades](/wiki/Difficulties) • [Hit Objects](/wiki/Hit_Objects) • [Técnicas de mapping](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
+[Beatmap](/wiki/Beatmaps) • [Hit object](/wiki/Hit_object) • [Mods](/wiki/Game_Modifiers) • [Pontuação](/wiki/Score) • [Replay](/wiki/Replay) • [Multi](/wiki/Multi)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# [Editor de beatmap](/wiki/Beatmap_Editor)
+
+Seções: [Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song setup](/wiki/Beatmap_Editor/Song_Setup)
 
-# Edição online e Aprovação
+Componentes: [AiMod](/wiki/Beatmap_Editor/AiMod) • [Beat snap divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [Distance snap](/wiki/Beatmap_Editor/Distance_Snap) • [Menu](/wiki/Beatmap_Editor/Menu) • [SB load](/wiki/Beatmap_Editor/SB_Load) • [Timelines](/wiki/Beatmap_Editor/Timelines)
 
-[Submissão](/wiki/Submission) • [Aprovação de Beatmaps](/wiki/Beatmap_ranking_procedure) • [Modding](/wiki/Modding) • [Ranking Criteria](/wiki/Ranking_Criteria): ([osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) •[osu!mania](/wiki/Ranking_Criteria/osu!mania))
+[Beatmapping](/wiki/Beatmapping) • [Dificuldades](/wiki/Difficulties) • [Técnicas de mapping](/wiki/Mapping_Techniques) • [Storyboarding](/wiki/Storyboarding)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# Envio de beatmap e avaliação
 
-# Mundo de osu!
+[Envio](/wiki/Submission) • [Modding](/wiki/Modding) • [Procedimento para ranqueado](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
 
-[Glossário](/wiki/Glossary) • [Torneios](/wiki/Tournaments) • [Skinning](/wiki/Skinning) • [Projetos](/wiki/Projects)
+[Ranking criteria](/wiki/Ranking_Criteria): [osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) • [osu!mania](/wiki/Ranking_Criteria/osu!mania)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# Comunidade
+
+[Torneios](/wiki/Tournaments) • [Skinning](/wiki/Skinning) • [Projetos](/wiki/Projects) • [Guias](/wiki/Guides) • [Servidor do Discord osu!dev](/wiki/osu!dev_Discord_server) • [Como você pode ajudar](/wiki/How_You_Can_Help!) • [Glossário](/wiki/Glossary)
```

Comment:
```suggestion
[Torneios](/wiki/Tournaments) • [Skinning](/wiki/Skinning) • [Projetos](/wiki/Projects) • [Guias](/wiki/Guides) • [Servidor de Discord do osu!dev](/wiki/osu!dev_Discord_server) • [Como você pode ajudar](/wiki/How_You_Can_Help!) • [Glossário](/wiki/Glossary)
```
Since that one got merged first, suggesting for consistency.